CHANGELOG.md

```
@@ -1,5 +1,10 @@
 # ETERNA Changelog
 
+## v1.0.1 (unreleased)
+
+#### Bug fixes
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)